server/sonar-server/src/test/java/org/sonar/server/user/ws/CurrentActionTest.java

```diff
@@ -65,16 +67,23 @@ public void return_user_info() {
       .setExternalIdentity("obiwan")
       .setExternalIdentityProvider("sonarqube")
       .setScmAccounts(newArrayList("obiwan:github", "obiwan:bitbucket"))
-      .setOnboarded(false));
+      .setOnboarded(false)
+      .setHomepageType("MY_PROJECTS"));
```

Comment:
no value if type if `MY_PROJECT` because it's only the My Projects page without any filtering?

Reply:
indeed